create_lmodsitepackage.py

```diff
@@ -19,6 +19,71 @@
     return content
 end
 
+local function load_sitespecific_hooks()
```

Comment:
typo here, doesn't match with what's below, which triggers:

```
/cvmfs/software.eessi.io/versions/2023.06/compat/linux/x86_64/usr/bin/lua5.1: /tmp/vsc40023/software-layer/.lmod/SitePackage.lua:153: variable 'load_site_specific_hooks' is not declared
stack traceback:
        [C]: in function 'error'
        ...ux/x86_64/usr/share/Lmod/libexec/../tools/strict.lua:36: in function <...ux/x86_64/usr/share/Lmod/libexec/../tools/strict.lua:34>
        /tmp/vsc40023/software-layer/.lmod/SitePackage.lua:153: in main chunk
        [C]: in function 'require'
        ...compat/linux/x86_64/usr/share/Lmod/libexec/utils.lua:1101: in function 'initialize_lmod'
        ...06/compat/linux/x86_64/usr/share/Lmod/libexec/ml_cmd:125: in function 'main'
        ...06/compat/linux/x86_64/usr/share/Lmod/libexec/ml_cmd:373: in main chunk
        [C]: ?
```

```suggestion
local function load_site_specific_hooks()

Reply:
Fixed